README.md

```diff
@@ -1,2 +1,115 @@
-# How-to-wrap-the-text-of-segment-in-SfSegmentedControl
-To have a multi line support in Xamarin.Forms SfSegmentedControl
+# Getting Started with Xamarin Segmented Control (SfSegmentedControl)
+
```

Comment:
@SwethavasanthSF3977  Update read me file related to wrap text

Reply:
Updated related file